buildSrc/build.gradle

```diff
@@ -47,6 +47,8 @@ if (project == rootProject) {
 Properties props = VersionPropertiesLoader.loadBuildSrcVersion(project.file('version.properties'))
```

Comment:
Maybe make a `File versionProperties = project.file('version.properties')` variable so it is obvious we reference it twice.